benchmark/csharp/Tpch/VectorFunctions.cs

```diff
@@ -3,49 +3,31 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using Apache.Arrow;
+using System.Collections.Generic;
+using Microsoft.Data.Analysis;
 
 namespace Tpch
 {
     internal static class VectorFunctions
     {
-        internal static DoubleArray ComputeTotal(DoubleArray price, DoubleArray discount, DoubleArray tax)
+        internal static PrimitiveDataFrameColumn<double> ComputeTotal(PrimitiveDataFrameColumn<double> price, PrimitiveDataFrameColumn<double> discount, PrimitiveDataFrameColumn<double> tax)
         {
             if ((price.Length != discount.Length) || (price.Length != tax.Length))
             {
                 throw new ArgumentException("Arrays need to be the same length");
             }
 
-            int length = price.Length;
-            var builder = new DoubleArray.Builder().Reserve(length);
-            ReadOnlySpan<double> prices = price.Values;
-            ReadOnlySpan<double> discounts = discount.Values;
-            ReadOnlySpan<double> taxes = tax.Values;
-            for (int i = 0; i < length; ++i)
-            {
-                builder.Append(prices[i] * (1 - discounts[i]) * (1 + taxes[i]));
-            }
-
-            return builder.Build();
+            return (PrimitiveDataFrameColumn<double>)(price * (1 - discount) * (1 + tax));
```

Comment:
Do we not have operator overloads for `PrimitiveDataFrameColumn<T>` that return the same type? I think that would make this code simpler so the user doesn't have to cast.

Reply:
No unfortunately. I'll file a bug.